internal/add/tui_test.go

```
@@ -0,0 +1,520 @@
+		Projects: map[string]config.Project{},
+	}
+	return NewAddModel(AddModelOptions{
+		WsRoot:    wsRoot,
+		Workspace: ws,
+		Save:      func(*config.Workspace) error { return nil },
+		Sources:   sources,
+	})
+}
+
+// staticSource always returns the same suggestions.
+type staticSource struct {
```

golangci-lint (2.11.4)

[error] type staticSource is unused

(unused)